src/django_remake_migrations/management/commands/remakemigrations.py

```diff
@@ -18,6 +18,21 @@
 from django_remake_migrations.conf import app_settings
 
 
+class CustomMigrationWriter(MigrationWriter):
+    """Custom MigrationWriter which adds support for run_before."""
+
+    def as_string(self) -> str:
+        """Add run_before if available."""
+        text = super().as_string()
+        if self.migration.run_before:
+            run_before_string = f"run_before = {self.migration.run_before}"
+            text = text.replace(
+                "class Migration(migrations.Migration):",
+                f"class Migration(migrations.Migration):\n    {run_before_string}",
+            )
+        return text
```

Comment:
Trying to add support for this in Django itself... So in the future this could be removed 
- https://code.djangoproject.com/ticket/36274
- https://github.com/django/django/pull/19303